### ISSUE TITLE
Fails WCAG standards

### ISSUE THREAD

**Issue by Some-of-the-things** [andyvr/picEdit#78]
The text in the image previous does not meet WCAG accessibility standards, and many labels do not have values, which is a further violation of WCAG standards.
